contracts/command-batcher/Cargo.toml

```diff
@@ -40,6 +40,12 @@ thiserror = { version = "1.0.31" }
 serde_json = "1.0.89"
 either = "1.8.1"
 hex = { version = "0.4.3", default-features = false, features = [] }
+connection-router = { workspace = true, features = ["library"] }
+ethabi = { version = "18.0.0", default-features = false, features = [] }
+sha3 = { version = "0.10", default-features = false, features = [] }
+multisig = { workspace = true, features = ["library"] }
+axelar-wasm-std = { workspace = true }
+k256 = { version = "0.11.1", features = ["ecdsa"] }
```

Comment:
this is what we discussed in a standup some time ago, first I tried this crate in the multisig contract: https://crates.io/crates/libsecp256k1 which is the one being used by the NEAR gateway but the build size when using it is much greater than the limit allowed. I ended up using `k256` because that's the one being used by cosmwasm itself: https://github.com/CosmWasm/cosmwasm/blob/0dae968db91734045239f45e982cddb1e5f11115/packages/crypto/Cargo.toml#L22.
There is another crate that seems promising but I haven't tried it yet: https://crates.io/crates/secp256k1 which is a wrapper of the C implementation used by bitcoin, not sure if there is an easy way to verify the implementation hasn't been modified in any way before being wrapped
I think we should keep it this way for the devnet deployment and circle back in another iteration. Thoughts?

Reply:
task to keep track of this: https://axelarnetwork.atlassian.net/browse/AXE-1366